src/Spatial/Euclidean/Polygon2D.cs

```diff
@@ -133,7 +133,7 @@ public static Polygon2D GetConvexHullFromPoints(IEnumerable<Point2D> pointList)
             // Order the hull points by angle to the centroid
             Point2D centroid = Point2D.Centroid(hullPoints);
             Vector2D xAxis = new Vector2D(1, 0);
-            var results = (from x in hullPoints select new Tuple<Angle, Point2D>(centroid.VectorTo(x).AngleTo(xAxis), x)).ToList();
+            var results = (from x in hullPoints select new Tuple<Angle, Point2D>(centroid.VectorTo(x).SignedAngleTo(xAxis, true), x)).ToList();
```

Comment:
Could you explain why this is going to be slow? In any case sorting on absolute angle does not seem really useful to me. Maybe I could make this sorting optional?